hack/verify-staging-imports.sh

```diff
@@ -77,4 +77,11 @@ if grep -rq '// import "k8s.io/kubernetes/' 'staging/'; then
 	exit 1
 fi
 
+IN_CLUSTER_EXAMPLE=vendor/k8s.io/client-go/examples/in-cluster
+test -d "${IN_CLUSTER_EXAMPLE}" # make sure example is still there
```

Comment:
Let's just comment out that import (but keep the line there to show people how to use the plugin if needed)